app/services/api/charts/company.rb

```diff
@@ -0,0 +1,94 @@
+module Api
+  module Charts
+    class Company
+      def initialize(company)
+        @company = company
+      end
+
+      # Returns array of following series:
+      # - company emissions
+      # - company's sector average emissions
+      # - company's sector CP benchmarks (scenarios)
+      #
+      # @example
+      #   [
+      #     { name: 'Air China',                   data: {'2014' => 111.0, '2015' => 112.0 } },
+      #
+      #     { name: 'Airlines sector mean',        data: { ... } },
+      #
+      #     { name: '2 Degrees (Shift-Improve)',   data: { ... } },
+      #     { name: '2 Degrees (High Efficiency)', data: { ... } },
+      #   ]
+      #
+      def emissions_data
+        [
+          emissions_data_from_company,
+          emissions_data_from_sector,
+          emissions_data_from_sector_benchmarks
+        ].flatten
+      end
+
+      private
+
+      def emissions_data_from_company
+        {
+          name: @company.name,
+          data: @company.cp_assessments.last.emissions
+        }
+      end
+
+      def emissions_data_from_sector
+        {
+          name: "#{@company.sector.name} sector mean",
+          data: sector_average_emissions
+        }
+      end
+
+      def emissions_data_from_sector_benchmarks
+        @company.sector_benchmarks.map do |benchmark|
```

Comment:
for the chart sector benchmarks should be only for the last date less than CP::Assessment date. 
Ex. 
if we have benchmarks available for 04.2017 and 05.2018, and assessment date is 06.2018 we take 05.2018 benchmarks
if we have the same benchmarks but the assessment date is 06.2017 we take benchmarks for 04.2017

Reply:
ok, so few thoughts on this:

1) the current implementation is clearly wrong because we can have multiple CP benchmarks per scenario and we don't want to see two or three `Paris Pledges` series on a single chart.

2) **CP Company chart** on mockups has `Assessment Date` select input which provides scope for company emissions series ... but does it necessarily mean that we shouldn't always show the latest CP Benchmarks? I kind of understand what's @tsubik point here (that sometimes it may be useful to show benchmarks from before last assessment) but want to confirm that.

3) the previous point is only about the **CP Company chart**. For **CP Sector chart** we don't have a single assessment context so we will always show most recent CP benchmarks, right?

/cc @simaob @agwozdowski @rapzo

Reply:
I've implemented what @tsubik suggested in https://github.com/Vizzuality/laws_and_pathways/pull/75 for **CP Company chart** (also fixed CP Sector chart there).

Reply:
fixed in #75